cmd/zero-trust-workload-identity-manager/main.go

```diff
@@ -38,6 +39,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 
 	operatoropenshiftiov1alpha1 "github.com/openshift/zero-trust-workload-identity-manager/api/v1alpha1"
+	spireServerController "github.com/openshift/zero-trust-workload-identity-manager/pkg/controller/spire-server"
```

Comment:
As '-' generally is used with command line options, it is not a recommended method to use them in filename or package names.
https://google.github.io/styleguide/go/decisions#naming